.travis.yml

```diff
@@ -8,10 +8,14 @@ install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
   - go get -u golang.org/x/net/context
+  - go get -u github.com/HewlettPackard/gas
   - export GO15VENDOREXPERIMENT=1
   - glide install
 
 script:
+  - gas -skip=*examples* -skip=*vendor* ./... | tee gas-scan.txt
```

Comment:
for some reason it's still not skipping. maybe try `-skip=management/examples/*.go`?